tests/integration/istioio/trafficmanagement/scripts/request_timeouts.txt

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Step 1: setup bookinfo & sleep pods
+kubectl label namespace default istio-injection=enabled --overwrite || true
+
+kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
```

Comment:
might be worth it to move this bookinfo stuff into it's own file.

Reply:
Yes, that will be a good idea to keep it separate and include it wherever required.

Reply:
So I'm thinking to use following structure to separate common functionality into its own folder.
istioio
 - common
    - scripts
      - bookinfo.txt
      - other_common_func.txt
 - examples
 - trafficmanagement
 - security

In this way if the `security` or `trafficmanagement` needs bookinfo they can import it from `common` folder.
Question: Whether this new `common` folder is appropriate or do you have any other suggestion.

Reply:
Using above mentioned folder structure for common functionality.